### DISCUSSION TITLE
Does this support monorepo structures?

### DISCUSSION THREAD

**Discussion by PromH** [semantic-release-cargo/semantic-release-cargo#244] [Q&A] [answered]
I have a repository that's structured as a monorepo (as outlined in this page of the docs: https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html).
Will the CLI be compatible with a monorepo structure?

**Comment by EricCrosson** [comment, marked as answer]
Yes, semantic-release-cargo supports workspaces. However, note that it will synchronize the version in each crate in your workspace. semantic-release-cargo does not support versioning each crate independently.

**Reply by PromH** [reply]
Awesome!
